workspaces/libnpmpublish/lib/publish.js

```diff
@@ -137,6 +139,13 @@ const buildMetadata = async (registry, manifest, tarballData, spec, opts) => {
     length: tarballData.length,
   }
 
+  if (provenance === true && provenanceBundle) {
+    throw Object.assign(
+      new Error('--provenance and --provenance-bundle cannot be specified at the same time.'),
+      { code: 'EUSAGE' }
+    )
+  }
```

Comment:
this seems confusing - if they're mutually exclusive, why not `--provenance=path/to/bundle`?

Reply:
I did try this. However, I wanted to preserve the existing `--provenance` boolean, and npm seems finicky about its flags. If I create a flag that can be a boolean or string then `npm publish --provenance package-name` will also no longer work and parse `package-name` as the argument to `--provenance`.